docs/source/configuration/telemetry/instrumentation/selectors.mdx

```diff
@@ -30,66 +30,79 @@ Each service of the router pipeline (`router`, `supergraph`, `subgraph`) has its
-| `response_status`  | Yes         | `code`\|`reason`                          | The response status                  |
-| `response_context` | Yes         |                                           | The name of a response context key   |
-| `baggage`          | Yes         |                                           | The name of a baggage item           |
-| `env`              | Yes         |                                           | The name of an environment variable  |
-| `on_graphql_error` | No          | `true`|`false`                            | Boolean set to true if the response payload contains a graphql error  |
-| `static`           | No          |                                           | A static string value                |
-| `error`            | No          | `reason`                                  | a string value containing error reason when it's a critical error                |
+| Selector              | Defaultable | Values                      | Description                                                          |
+|-----------------------|-------------|-----------------------------|----------------------------------------------------------------------|
+| `trace_id`            | Yes         | `open_telemetry`\|`datadog` | The trace ID                                                         |
+| `operation_name`      | Yes         | `string`\|`hash`            | The operation name from the query                                    |
+| `studio_operation_id` | Yes         | `true`\|`false`             | The Apollo Studio operation id                                       |
+| `request_header`      | Yes         |                             | The name of the request header                                       |
+| `response_header`     | Yes         |                             | The name of a response header                                        |
+| `response_status`     | Yes         | `code`\|`reason`            | The response status                                                  |
+| `response_context`    | Yes         |                             | The name of a response context key                                   |
+| `baggage`             | Yes         |                             | The name of a baggage item                                           |
+| `env`                 | Yes         |                             | The name of an environment variable                                  |
+| `on_graphql_error`    | No          | `true`\|`false`             | Boolean set to true if the response payload contains a graphql error |
+| `static`              | No          |                             | A static string value                                                |
+| `error`               | No          | `reason`                    | a string value containing error reason when it's a critical error    |
 
 #### Supergraph
```

Comment:
These should all be H3s to they're visible in the table of contents. Again, happy to update.

Reply:
Yes, go ahead

Reply:
https://github.com/apollographql/router/pull/5472/commits/f5153df05b8b984ef5d9e1fe3b1b08344bc6d03f